[DM-2981] Breadcrumps added to all device tab (#1723)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-10-18-497-0-breadcrumps-added-to-all-device-tab.md

---
date: ""

> title: Breadcrumps added to all device tab

Reviewer: ```suggestion
title: Breadcrumps added to all device details tabs
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-2981
version: 10.18.497.0
---

> On device details page, breadcrumbs have been added to every tab.

Reviewer: ```suggestion
In the device details, breadcrumbs have been added to every tab.
```
